Cargo.toml

```
@@ -17,7 +17,7 @@ edition = "2021"
 rust-version = "1.86.0"
 
 [workspace.dependencies]
-arrow = { version = ">=56.0", default-features = false }
-arrow-array = { version = ">=56.0", default-features = false }
-parquet = { version = ">=56.0", default-features = false, features = ["encryption"] }
+arrow = { version = "~56", default-features = false }
+arrow-array = { version = "~56.0", default-features = false }
+parquet = { version = "~56.0", default-features = false, features = ["encryption"] }
```

I think we should do this instead, which will accept any 56.x release of arrow/parquet. Your proposed change will use 56.0 instead of the more recent 56.2 which is also compatible.

```suggestion
arrow = { version = "56.0", default-features = false }
arrow-array = { version = "56.0", default-features = false }
parquet = { version = "56.0", default-features = false, features = ["encryption"] }
```